OpenTAP.TUI/Views/TestPlanView.cs

```diff
@@ -380,13 +380,39 @@ public void SaveTestPlan(string path)
                 Application.Run(dialog);
                 if (dialog.FileName != null)
                     path = Path.Combine(dialog.DirectoryPath.ToString(), dialog.FilePath.ToString());
+
+                if (File.Exists(path))
+                {
+                    // Open dialog to ask the user if they want to override the file.
+                    if (MessageBox.Query("Override existing file?", $"Are you sure you want to override\n{Path.GetFileName(path)}", "Override", "Cancel") == 1)
```

Comment:
This should be overwrite, right?

```suggestion
                    if (MessageBox.Query("Override existing file?", $"Are you sure you want to override\n{Path.GetFileName(path)}", "Override", "Cancel") == 1)
```